frontend/apps/reader/modules/readerfont.lua

```diff
@@ -405,6 +428,266 @@ function ReaderFont:onDecreaseFontSize(ges)
     return true
 end
 
+local font_family_info_text = _([[
+In HTML/CSS based documents like EPUBs, stylesheets can specify to use fonts by family name instead of a specific font name.
+Except for monospace and math, KOReader uses your default font for any family name.
+You can associate a specific font to each family name if you care about the distinction.
+Long-press on a font name will make the association global (★), so it applies on all your books. You should set a global association if you want to notice a book use font families.
+Tap will make the association for the current book only.
```

Comment:
nit: Ditto, `A tap` or `Tapping`.

Reply:
I'd possibly simplify to `will only affect the current book.`

"make the association" feels a bit clunky here (but less so above?).